README.md

```diff
@@ -31,72 +42,120 @@ _Kev_ reduces the need for Kubernetes expertise in the team. The generated Kuber
 
 * **Easy integrations** - You may use generated Kubernetes manifests with any tool / framework of your choice. We aim at adding some useful integrations further improving developer experience.
 
-## Quick Start
+## Installation
 
-All you need to get started quickly is [kev](https://github.com/appvia/kev/releases) binary added to your PATH, and one or more docker compose files.
+All you need to get started quickly is the [kev](https://github.com/appvia/kev/releases) binary added to your PATH, and one or more docker compose files.
 
-### Initiate a project
+## Quickstart
 
-Run the following command from within your project directory to initialise Kev project:
+- `kev init` - identifies a project's Compose Kubernetes source files and creates Compose environment overrides.
+- `kev render` - detects, applies any config changes and generates deployment manifests.
+- `kev help` - run it if you're a little lost.
+
+### init
```

Comment:
tiny one: make `init` and `render` (below) bold so they stand out more.

Reply:
Reverted to older suggestions. They actually work better.